src/otx/core/model/entity/detection.py

```diff
@@ -84,48 +93,102 @@ def _export_parameters(self) -> dict[str, Any]:
                 ("model_info", "iou_threshold"): str(0.5),
             },
         )
+        parameters["additional_output_names"] = ("saliency_map") if self.explain_mode else ()
```

Comment:
OTXModelExporter already contains `output_names` parameter in constructor. Can we use it (like in classificaiton)?